### DISCUSSION TITLE
Planning engine terminating due to segmentation violation

### DISCUSSION THREAD

**Discussion by sgfisme** [frePPLe/frepple#425] [Q&A]
Hi, 
I tried to build my supply path on the frePPLe cloud, but I got the error result when creating a plan. The error log is as below. Could you give me an idea for debugging?

Clusters: 48

Memory usage:
Model                 	Count	Memory
-----                 	-----	------
Plan                  	1	1097
Location              	28	5501
Customer              	17	3060
Supplier              	4	761
Buffer                	3934	1632575
Setup matrix          	0	0
Setup matrix rules    	0	0
Resource              	0	0
Skill                 	0	0
Resource skill        	0	0
Operation             	24643	11466623
Operation material    	6244	1154660
operation resource    	0	0
Calendar              	5	7328
Calendar buckets      	5649	740019
Item                  	747	2016895
Item suppliers        	48	11664
Item distributions    	673	150752
Demand                	880	330000
Constraints           	0	0
OperationPlan         	21688	7243260
OperationPlan material	4793	613504
OperationPlan resource	0	0
Problem               	21810	1918536
String pool           	906	65232
Memory cache          	0	0
Total                 		27361467

INFO Finished 'Calculate inventory parameters' at 11:27:30 

INFO Start step 181 'Export inventory parameters' at 11:27:30
INFO Finished 'Export inventory parameters' at 11:27:30 

INFO Start step 190 'Performing inventory rebalancing' at 11:27:30
INFO Number of generated balancing requests: 0
INFO Finished 'Performing inventory rebalancing' at 11:27:30 

INFO Start step 199 'Make initial plan feasible' at 11:27:30
INFO Erasing existing proposed plan
INFO Propagating work-in-progress status information
INFO Resolving current infeasibilities
Initial problems:
     unplanned: 880
INFO Finished 'Make initial plan feasible' at 11:27:30 

INFO Start step 200 'Generate supply plan' at 11:27:30
INFO Plan type: 1
INFO Constraints: 9
Planning engine terminating due to segmentation violation

Planning engine terminating due to segmentation violation

Planning engine terminating due to segmentation violation

Planning engine terminating due to segmentation violation

Planning engine terminating due to segmentation violation

Planning engine terminating due to segmentation violation

Planning engine terminating due to segmentation violation

**Comment by jdetaeye** [comment]
Set the parameter "plan.loglevel" to 9.  
The log file will get much more verbose, and the end of it should hopefully give you a clue where things go bad.

I'd be interested to hear what you find because such a crash shouldn't happen.  Even when you throw wrong data at the planning engine...

**Reply by sgfisme** [reply]
Thanks for your suggestion @jdetaeye . I tried to set the parameter "plan.loglevel" to 9, and created a plan successfully. I will keep watching and feedback to you if I got the same error.
